packages/block/src/index.ts

```diff
@@ -4,7 +4,7 @@ export * from './consensus/index.js'
 export { type BeaconPayloadJSON, executionPayloadFromBeaconPayload } from './from-beacon-payload.js'
 export * from './header/index.js'
 export {
-  genRequestsTrieRoot,
+  genRequestsRoot,
```

Comment:
Should we call this genRequestsHash ?

Reply:
Yep this is actually a better name since it is not a trie root anymore!